cmd/conf/init.go

```diff
@@ -74,6 +76,7 @@ func InitConfigAddOptions(prefix string, f *pflag.FlagSet) {
 	f.String(prefix+".dev-init-address", InitConfigDefault.DevInitAddress, "Address of dev-account. Leave empty to use the dev-wallet.")
 	f.Uint64(prefix+".dev-init-blocknum", InitConfigDefault.DevInitBlockNum, "Number of preinit blocks. Must exist in ancient database.")
 	f.Bool(prefix+".empty", InitConfigDefault.Empty, "init with empty state")
+	f.Bool(prefix+".extract-wasm", InitConfigDefault.ExtractWasm, "if set, extract the wasm directory when downloading a database (this can be security concern because it contains native executables)")
```

Comment:
Parenthesis not clear IMHO.. maybe something like
(contains executable code - only use with highly trusted source)
?